include/highfive/H5PropertyList.hpp

```diff
@@ -244,15 +259,29 @@ class FileVersionBounds {
         : _low(low)
         , _high(high) {}
 
+    FileVersionBounds(const FileAccessProps& fapl) {
+        if (H5Pget_libver_bounds(fapl.getId(), &_low, &_high) < 0) {
+            HDF5ErrMapper::ToException<PropertyException>("Unable to access file version bounds");
+        }
+    }
+
+    H5F_libver_t get_low() const {
```

Comment:
I think here we could produce a `std::pair`

Reply:
I did ;)